api/client/gro_client.py

```diff
@@ -82,6 +88,54 @@ def get_df(self):
                 self._data_frame = self._data_frame.merge(tmp, how='outer')
         return self._data_frame
 
+    def GDH(self, gdh_selection):
+        """Wrapper for get_data_points with alternative input and output style.
+
+        The selection of data series to retrieve is encoded in a
+        'gdh_seletion' string of the form
+        <metric_id>-<item_id>-<region_id>-<partner_region_id>-<source_id>-<frequency_id>
+
+        For example, client.GDH("860032-274-1231-0-14-9") will get the
+        data points for Production of Corn in China from PS&D at an
+        annual frequency, e.g.
+        for csv_row in client.GDH("860032-274-1231-0-14-9"):
+            print csv_row
+
+        Parameters:
+        ----------
+        gdh_selection: string
+
+        Returns:
+        ------
+        DataFrame
+
+            subset of the main DataFrame get_df() with only the requested series.
+
+        """
+        entity_keys = ['metric_id', 'item_id', 'region_id', 'partner_region_id',
```

Comment:
@JGrinst  @nemozen 
I added partner_region as required entity key , and included unit in output 

I'm think it could be a separate improvement to support user requested params like custom unit, revisions etc